_articles/getting-started/how-to-change-your-master-password.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: article
+title: How to change your master password
+category: Getting Started
+featured: false
+popular: false
+tags: [password, account]
+---
+
+The master password can only be changed from the [web](https://vault.bitwarden.com) vault.
+
+1. Log in to the [web vault](https://vault.bitwarden.com)
+2. Click **Settings** on the sidebar 
+3. Click **Change Master Password** under the **Master Password** section
+4. Type in your current password, the new password you want, and then re-type to confirm
+5. <span title="You will be logged out and redirected to the web vault login screen.">
+    Click **Change Password**
+   </span>
+    * <span style="color:red">**Note**</span>: Clicking **Change Password** will log you out of the web vault
```

Comment:
Let's try to keep HTML out of these if we can and rely on markdown as much as possible.